app/controllers/concerns/idv_session.rb

```diff
@@ -0,0 +1,59 @@
+module IdvSession
+  extend ActiveSupport::Concern
+
+  if Figaro.env.proofing_vendors
+    Figaro.env.proofing_vendors.split(/\W+/).each do |vendor|
+      vendor_path = "#{Rails.root}/vendor/#{vendor}/lib"
+      $LOAD_PATH.unshift vendor_path
+    end 
+  end 
+
+  protected
+
+  def idv_question_number
+    session[:idv_question_number] ||= 0
```

Comment:
From my recent investigations into use the of sessions with device, I would recommend using #user_session everywhere rather than #session.   In theory if we always sign_out correctly in should make any difference but we had issues where we only signed out the :user scope and then any variables in the top level session would persist between login sessions or different users.

Now that we have fixed that issue and we always sign_out without a limited scope i think its probably not an issue, but its probably still better/safer to store stuff in the user_session.